numpy/lib/nanfunctions.py

```diff
@@ -548,7 +551,7 @@ def nanprod(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
     Parameters
     ----------
     a : array_like
-        Array containing numbers whose sum is desired. If `a` is not an
+        Array containing numbers whose product is desired. If `a` is not an
         array, a conversion is attempted.
```

Comment:
`ptp`, `argmin`, `argmax`. Also, the following might benefit as well, but the internal order of the array would matter: `partition`, `argpartition`, `sort`, `argsort`, `searchsorted`, `cumsum`, `cumprod`. I was even thinking that it might be worth allowing `ravel` to do a partial ravel along a subset of the axes (basically like `_ureduce` does), but that is probably going to require some major rewriting to do properly.